_posts/2016-03-04-terrier-open-source-sensitive-democracy-for-hamilton-tles.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "TERRIER OPEN SOURCE SENSITIVE DEMOCRACY FOR HAMILTON TILES:  TGIF! (73)"
```

Comment:
no need to do all caps, I think it’s done with CSS on the website later. See past TGIF for comparison https://raw.githubusercontent.com/hoodiehq/hood.ie/gh-pages/_posts/2015-07-10-golden-robots-tgif-72.md